Source/WebKit/WebProcess/InjectedBundle/API/gtk/DOM/WebKitDOMHTMLDocument.cpp

```diff
@@ -299,18 +299,14 @@ glong webkit_dom_html_document_get_width(WebKitDOMHTMLDocument* self)
 {
     WebCore::JSMainThreadNullState state;
     g_return_val_if_fail(WEBKIT_DOM_IS_HTML_DOCUMENT(self), 0);
-    WebCore::HTMLDocument* item = WebKit::core(self);
-    glong result = item->width();
-    return result;
+    return 0;
```

Comment:
This should just become return 0.

Reply:
Done in [1f1978a](https://github.com/WebKit/WebKit/commit/1f1978a816967eb52e39cb17d27c6b03c576b9a3).